DualCore/cm4/HelloWorld_cm4.cproject.yml

```diff
@@ -24,33 +10,34 @@ project:
 
   add-path:
     - ../middleware/multicore/mcmgr/src
-    - ./RTE/CMSIS
 
   components:
-    - component: NXP::Device:Startup
-      warnings: off 
-    - component: NXP::Device:CMSIS:K32L3A60_header
-    - component: NXP::Device:CMSIS:K32L3A60_system
-
-    - component: NXP::Device:SDK Drivers:clock
-    - component: NXP::Device:SDK Drivers:common
-    - component: NXP::Device:SDK Drivers:gpio
-    - component: NXP::Device:SDK Drivers:i2c
-    - component: NXP::Device:SDK Drivers:lists
-    - component: NXP::Device:SDK Drivers:lpuart
-    - component: NXP::Device:SDK Drivers:lpuart_adapter
-    - component: NXP::Device:SDK Drivers:msmc
-    - component: NXP::Device:SDK Drivers:mu
-    - component: NXP::Device:SDK Drivers:port
-
-    - component: NXP::Device:SDK Utilities:assert
-    - component: NXP::Device:SDK Utilities:debug_console
-    - component: NXP::Device:SDK Utilities:serial_manager
-    - component: NXP::Device:SDK Utilities:serial_manager_uart
-      
-    - component: NXP::Board Support:SDK Project Template:project_template&K32L3A60
-
-    - component: ARM::CMSIS:CORE
+    - component: Device:Startup
+      warnings: off
+    - component: Device:CMSIS:K32L3A60_header
+    - component: Device:CMSIS:K32L3A60_system
+
+    - component: Device:SDK Drivers:clock
+    - component: Device:SDK Drivers:common
+    - component: Device:SDK Drivers:gpio
+    - component: Device:SDK Drivers:i2c
+    - component: Device:SDK Drivers:lists
+    - component: Device:SDK Drivers:lpuart
+    - component: Device:SDK Drivers:lpuart_adapter
+    - component: Device:SDK Drivers:msmc
+    - component: Device:SDK Drivers:mu
+    - component: Device:SDK Drivers:port
+
+    - component: Device:SDK Utilities:assert
+    - component: Device:SDK Utilities:debug_console
+    - component: Device:SDK Utilities:K32L3A60_linker
```

Comment:
from which component are the linker scripts contributed? Is it the template?

Reply:
K32L3A60_linker provided linker scripts but I've removed this and project now uses regions header file.